docker/entrypoint.sh

```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
 
-set -o errexit  
-set -o pipefail  
+set -o errexit
+set -o pipefail
 set -o nounset
 
-python manage.py collectstatic --noinput  
```

Comment:
these are not equivalent right?

Reply:
Do you mean the removed `makemigrations` lines? Those apps don't exist any more so I think it is outdated. Anyway `python manage.py makemigrations/migrate` should handle all.

The `collectstatic` line is kept below.